src/test/java/com/bbaga/githubscheduledreminderapp/domain/sources/github/PathFilterTest.java

```diff
@@ -0,0 +1,56 @@
+package com.bbaga.githubscheduledreminderapp.domain.sources.github;
+
+import com.bbaga.githubscheduledreminderapp.domain.configuration.sources.filters.PathFilterConfig;
+import com.bbaga.githubscheduledreminderapp.domain.sources.github.filters.PathFilter;
+import com.bbaga.githubscheduledreminderapp.infrastructure.github.GitHubIssue;
+import com.bbaga.githubscheduledreminderapp.infrastructure.github.GitHubPullRequest;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import java.io.IOException;
+import java.util.*;
+
+public class PathFilterTest {
```

Comment:
This is the only testing I've done - I have not done a full end-to-end test with a real config file.